pkg/plugins/scorer/load_aware_scorer.go

```diff
@@ -2,29 +2,44 @@ package scorer
 
 import (
 	"context"
+	"encoding/json"
+	"fmt"
 
-	"sigs.k8s.io/controller-runtime/pkg/log"
+	"sigs.k8s.io/gateway-api-inference-extension/pkg/epp/plugins"
 	"sigs.k8s.io/gateway-api-inference-extension/pkg/epp/scheduling/framework"
 	"sigs.k8s.io/gateway-api-inference-extension/pkg/epp/scheduling/types"
-	"sigs.k8s.io/gateway-api-inference-extension/pkg/epp/util/env"
 )
 
 const (
 	// LoadAwareScorerType is the type of the LoadAwareScorer
 	LoadAwareScorerType = "load-aware-scorer"
 
-	queueThresholdEnvName = "LOAD_AWARE_SCORER_QUEUE_THRESHOLD"
-	queueThresholdDefault = 128
+	// QueueThresholdDefault defines the default queue threshold value
+	QueueThresholdDefault = 128
 )
 
+type loadAwareScorerParameters struct {
+	Threshold float64 `json:"threshold"`
```

Comment:
should be `int`

Reply:
The threshold here is a float64. It was read from an environment variable and converted to a float64